gapir/client/session.go

```diff
@@ -295,10 +296,19 @@ func (s *session) newADB(ctx context.Context, d adb.Device, abi *device.ABI) err
 		return err
 	}
 
+	var completeLaunchArgs []string;
```

Comment:
no semicolons. Please use gofmt to format your code (it's a good idea to get your IDE to format-on-save).

Also, this can be initialized with all the fields in one declaration:

```go
completeLaunchArgs := []string{
	"--idle-timeout-sec", string(int(sessionTimeout /time.Second)),
	"--auth-token-file", string(s.auth),
	"--android",
}
```

Reply:
Whoops. I do this in python all the time. It's a bad habit from being a c++ centric programmer.

Reply:
Good tip with the single initialiser. I only started go yesterday so while obvious, this didn't cross my mind.